fofix/core/Config.py

```diff
@@ -66,6 +73,8 @@ def read(self, filenames):
             try:
                 with open(filename) as fp:
                     configData = fp.read()
+
+                configData = decodeString(configData)
```

Comment:
Is [``fretwork.unicodify``](https://github.com/fofix/fretwork/blob/5dcbfd148ceef5206f55106037bbfbe9bac75f4d/fretwork/unicode.py#L35) working ? Because it could be useful not to add a basic function to this big code ;)

Reply:
Nope, not enough for this case.